src/Kernel32/Kernel32.cs

```diff
@@ -157,6 +157,14 @@ public static partial class Kernel32
         [DllImport(api_ms_win_core_processthreads_l1_1_1)]
         public static extern int GetCurrentProcessId();
 
+        /// <summary>
+        /// Retrieves the process identifier of the specified process.
+        /// </summary>
+        /// <param name="Process">A handle to the process. The handle must have the PROCESS_QUERY_INFORMATION or PROCESS_QUERY_LIMITED_INFORMATION access right. For more information, see <see cref="https://msdn.microsoft.com/de-de/library/windows/desktop/ms684880(v=vs.85).aspx"/></param>
+        /// <returns>The process identifier of the specified process.</returns>
+        [DllImport(api_ms_win_core_processthreads_l1_1_1)]
+        public static extern int GetProcessId(IntPtr Process);
```

Comment:
You will need to add this method to the PublicAPI.Nonshipping.txt file for the project. There should be a code fix offered if you click the light bulb on this line to automatically make this change.

Reply:
Light Bulb doesnt show anything useful up to me, NonShipping.txt does also not exist, do you mean the Unshipped.txt?

![Image of VS](https://i.imgur.com/2dif5Ct.png)

Reply:
Close. But I think you need to change `IntPtr` to `System.IntPtr` in the unshipped.txt file.

Reply:
Ahh, okay, hehe, thanks for your patience :D

Reply:
No problem. I appreciate your contribution.

There's still the trailing `"` symbol that doesn't belong. 😲